app/models/agents/imap_folder_agent.rb

```diff
@@ -55,6 +57,14 @@ class ImapFolderAgent < Agent
 
       Set `mark_as_read` to true to mark found mails as read.
 
+      Set `event_headers` to a list of header names you want to include in a `headers` hash in each created event, either in an array of string or in a comma-separated string.
```

Comment:
Nit: Since this is now supported by multiple Agents can we have this part of the description in the concern or is it sightly different per Agent?

Reply:
Alright, I thought we already had more than two 👍